invenio_rdm_records/templates/semantic-ui/invenio_notifications/community-submission.submitted.jinja

```diff
@@ -0,0 +1,24 @@
+{# notifications/community-submission.submitted.jinja #}
+
+{%- block subject -%}
+New record submission for your community {{ notification.context.get("request").get("receiver").get("metadata").get("title") }} submitted by {{ notification.context.get("request").get("created_by").get("username") }}
+{%- endblock subject -%}
+
+{%- block html_body -%}
```

Comment:
shelve: not for now, but we should explore how to address email template headers/footers and overall styling e.g. via extending some base templates.

Reply:
This is what I was suggesting when we had the demo.